src/packages/cli/src/__tests__/integrate.sqlite.test.ts

```diff
@@ -1885,18 +1797,100 @@ function tests(): Test[] {
           \`unsupported\` binary(50) DEFAULT NULL
         );
       `,
-      down: `
+    down: `
         drop table if exists column_name_that_becomes_empty_string;
         drop table if exists invalid_enum_value_name;
         drop table if exists no_unique_identifier;
         drop table if exists unsupported_type;
       `,
-      do: async (client) => {
-        return await client.column_name_that_becomes_empty_string.findMany({})
-      },
-      expect: [],
+    do: async (client) => {
+      return await client.column_name_that_becomes_empty_string.findMany({})
     },
-  ]
+    expect: [],
+  },
+]
+
+it.each(prepareTestKases(testKases))('%s', async (name, kase) => {
```

Comment:
I tried for a little while to get [`it.concurrent.each`](https://jestjs.io/docs/en/api#testconcurrenteachtablename-fn-timeout) to work. But it doesn't seem to work––snapshots keep getting errors. And each runs leads to different snapshot errors. I think it is related to https://github.com/facebook/jest/issues/2180 but we're explicitly naming our snapshots here so I don't really understand _why_ it would fail... Its too bad because I think it could really help us here. Current time is 44s. I'm curious how low we could get this...